README.rst

```diff
@@ -0,0 +1,47 @@
+==================================
+Spilo: HA PostgreSQL Clusters with Docker
+==================================
+
+Spilo is a Docker image that provides PostgreSQL and `Patroni <https://github.com/zalando/patroni>`__ bundled together. Patroni is a template for PostgreSQL HA. Multiple Spilos can create a resilient High Available PostgreSQL cluster. For this, you'll need to start all participating Spilos with `etcd <https://github.com/coreos/etcd>`__ and cluster name parameters. Spilo's name derives from სპილო [spiːlɒ], the Georgian word for "elephant."  
```

Comment:
not just with etcd and cluster name parameters, but with identical ones, i.e. all Spilos participating in a single HA cluster should point to the same Etcd and have the same cluster name (which is called scope in Spilo).